test/parallel/test-stream-readable-with-unimplemented-_read.js

```diff
@@ -5,4 +5,4 @@ const assert = require('assert');
 
 const readable = new stream.Readable();
 
-assert.throws(() => readable.read(), /not implemented/);
+assert.throws(() => readable.read(), /ERR_STREAM_READ_NOT_IMPLEMENTED/);
```

Comment:
can you use `common.expectsError` here as well?

Reply:
I've updated this test